### PR TITLE
Add "Sanitize media files" page and the correspondent reducers/API

### PR DESCRIPTION
**Pull request by davidmz**

### CODE REVIEW

File: src/components/settings/forms/privacy.jsx [hunk 1]
```diff
@@ -92,6 +93,9 @@ export default (function PrivacyForm() {
             </label>
           </div>
         </div>
+        <p className="text-muted">
+          Process the previously uploaded files <Link to="/settings/sanitize-media">here</Link>.
```
Comment (n1313) [inline src/components/settings/forms/privacy.jsx]: `Sanitize previously uploaded files here`?

File: src/components/settings/sanitize-media.jsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      <section className={styles.formSection}>
+        <p>
+          Images and other media might contain sensitive metadata such as geographical coordinates
+          of the place where they were taken, serial number of lens/camera and even your name. We do
+          our best to remove this data automatically, in case you don&#x2019;t want to make it
+          public.
+        </p>
+        {sanitizeMediaMetadata ? (
+          <p>
+            The option which makes sure that NEW files are automatically sanitized is enabled for
```
Comment (n1313) [inline src/components/settings/sanitize-media.jsx]: `New files that you upload are sanitized automatically, but ...`

File: src/components/settings/sanitize-media.jsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                {pluralForm(toProcess, 'older file', null, 'w')} which still might contain this
+                info.
+              </>
+            ) : (
+              '.'
+            )}
+          </p>
+        ) : (
+          <p>
+            The option which makes sure that NEW files are automatically sanitized is disabled for
```
Comment (n1313) [inline src/components/settings/sanitize-media.jsx]: `New files that you upload are <strong>not</strong> sanitized automatically. You can enable it ...`

File: src/components/settings/sanitize-media.jsx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+function SanitizeTask({ task }) {
+  const dispatch = useDispatch();
+  const status = useSelector((state) => state.sanitizeMediaStatus);
+
+  const submit = useCallback(() => dispatch(sanitizeMedia()), [dispatch]);
+
+  if (task) {
+    return (
+      <p>
+        <Icon icon={faClock} /> The sanitation process was started at{' '}
```
Comment (n1313) [inline src/components/settings/sanitize-media.jsx]: `The sanitization process...` (https://wikidiff.com/sanitization/sanitation)

File: src/components/settings/sanitize-media.jsx (hunk 3 above)
Comment (davidmz) [reply src/components/settings/sanitize-media.jsx]: Тут, наверное, не “enable it”? Не очень понятно, что за it.

Comment (n1313) [reply src/components/settings/sanitize-media.jsx]: Можно `You can change/toggle/control this ...`, или (хуже) `You can enable this behavior...`

**Comment by davidmz** [comment]
Обновил все тексты

**Review by n1313 — APPROVED** [review]

**Pull request merged**